_datafiles/templates/help/merchant.template

```diff
@@ -0,0 +1,10 @@
+<ansi fg="black-bold">.:</ansi> <ansi fg="magenta">Additional help for </ansi><ansi fg="command">merchant</ansi>
+
+Merchants have 2 skills assigned to it and is one of the jobs available to 
+players.  The following skills are available to Merchants:
+
+-Peep
+-Trading
+
+
+<ansi fg="magenta-bold">See also:</ansi> <ansi fg="command">help job</ansi>
```

Comment:
is `job` a helpfile? I see jobs.template, which should be `help jobs` unless there is an alias defined in the keywords file...

Reply:
my bad i just forgot to put an s at the end of job, I did one file, then copy and pasted for the rest of the files just changing the job name to match the job file